1회차/이동현/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,18 @@
 import sys
 
 sys.stdin = open("_모음이보이지않는사람.txt")
+
+moum = 'aeiou'
+
+TC = int(input())
+result = ''
+
+for _ in range(1, TC + 1):
+    sentence = input()
+    for i in range(len(sentence)):
+        if sentence[i] in moum:
+            continue
+        else:
+            result += sentence[i]
```

Comment:
좋은 풀이군요! 😄 `continue`를 사용하지 않은 풀이도 있습니다!
```suggestion
        if sentence[i] not in moum:
            result += sentence[i]
```